<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>601</width>
     <height>303</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDPasienLabel">
      <property name="text">
       <string>ID Pasien</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="EditPasien"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaPasienLabel">
      <property name="text">
       <string>Nama Pasien</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="EditNama"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="alamatLabel">
      <property name="text">
       <string>Alamat</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EditAlamat"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="jenisKelaminLabel">
      <property name="text">
       <string>Jenis Kelamin</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cmbJekel">
      <item>
       <property name="text">
        <string>Laki - Laki</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Perempuan</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="tanggalLahirLabel">
      <property name="text">
       <string>Tanggal Lahir</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="tempatLahirLabel">
      <property name="text">
       <string>Tempat Lahir</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="EditLahir"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="noHandPhoneLabel">
      <property name="text">
       <string>No HandPhone</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="EditPhone"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="jenisPasienLabel">
      <property name="text">
       <string>Jenis Pasien</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="cmbPasien">
      <item>
       <property name="text">
        <string>BPJS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Umum</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="EditDate"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>771</width>
     <height>192</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>96</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID Pasien</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama Pasien</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Alamat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jenis Kelamin</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tanggal Lahir</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tempat Lahir</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>No HandPhone</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jenis Pasien</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
